pandas/tseries/offsets.py

```diff
@@ -516,12 +402,11 @@ def nanos(self):
 
 
 class SingleConstructorOffset(DateOffset):
-
     @classmethod
     def _from_name(cls, suffix=None):
         # default _from_name calls cls with no args
         if suffix:
-            raise ValueError("Bad freq suffix {suffix}".format(suffix=suffix))
+            raise ValueError("Bad freq suffix %s" % suffix)
```

Comment:
revert, we are moving towards new style string formatting

Reply:
Woops, copy/paste from an older version.  Will revert.